editor/controllers/horizontal_controller.js

```diff
@@ -98,7 +98,8 @@ angular.module('ftbApp', []).controller('ftbQuestionFormController', ['$scope',
   }
   $scope.formValidation = function() {
     $scope.submitted = true;
-    var formValid = $scope.ftbForm.$valid && /\[\[.*?\]\]/g.test($scope.ftbFormData.question.text);
+    var ftbFormValid = $scope.ftbFormData.question.text.length;
+    var formValid = (ftbFormValid > 0) && /\[\[.*?\]\]/g.test($scope.ftbFormData.question.text);
```

Comment:
`ftbFormValid` has a numeric value? Why the comparison `> 0`?

Reply:
Yes numeric, refer above line 101. Checking question length.

Reply:
:) Can we please rename the variable to `questionTextLength` or something?